packages/astroport/Cargo.toml

```diff
@@ -19,6 +19,7 @@ backtraces = ["cosmwasm-std/backtraces"]
 cw20 = { version = "0.8" }
 terra-cosmwasm = { version = "2.2.0" }
 cosmwasm-std = { version = "0.16.2", features = ["iterator"] }
+cosmwasm-bignumber = "2.2.0"
```

Comment:
not sure how to eliminate this. I just took the response type from anchor.

Reply:
Use Anchor's crate and remove this dependency

Reply:
How can I do that? I'm quite new to smart contract development.

Reply:
```anchor-moneymarket = {git = "https://github.com/Anchor-Protocol/money-market-contracts", tag = "v0.3.1", package = "moneymarket"}```

I didn't test it but Cargo.toml entry should be something like this.

Reply:
I used this now:
moneymarket = { version = "0.3.0" }